### PR TITLE
Update comment and status text

### PR DESCRIPTION
**Pull request by dkwon17**
Signed-off-by: David Kwon <dakwon@redhat.com>

### Description
Updates action comment from:`Click here to try in Web IDE:` to `Click here to try the PR in web IDE:`
Updates action status from: `Open in Web IDE` to `Review and test in web IDE`

Here is a screenshot showing the new changes:
image

### Checklist

- [ ] This PR includes a documentation change
- [ ] This PR does not need a documentation change
---
- [x] This PR includes test changes
- [ ] This PR's changes are already tested
---
- [x] This change is a patch change
- [ ] This change is a minor change
- [ ] This change is a major (breaking) change

### CODE REVIEW

**Comment by dkwon17** [comment]
If the new texts look good, I can also update the README with new screenshots in this PR

File: src/logic/handle-pull-request-logic.ts [hunk 1]
```diff
@@ -53,7 +53,7 @@ export class HandlePullRequestLogic implements Logic {
         targetUrl: string,
         badgeUrl: string,
     ): Promise<void> {
-        const comment = `Click here to try in Web IDE: [![Contribute](${badgeUrl})](${targetUrl})`;
+        const comment = `Click here to try the PR in web IDE: [![Contribute](${badgeUrl})](${targetUrl})`;
```
Comment (ibuziuk) [inline src/logic/handle-pull-request-logic.ts:56]: @dkwon17 for the sake of consistency let's also change the comment to `Review and test in web IDE`

Comment (ibuziuk) [reply src/logic/handle-pull-request-logic.ts:56]: also, just a note that `here` in the docs / links is considered to be a bad practice - many style guides for technical writing (IBM Stylguide, DITA Best Practices) discourage the usage.

Comment (ibuziuk) [reply src/logic/handle-pull-request-logic.ts:56]: after some discussion, it was decided to leave it `Click here to review and test in web IDE`

**Review by ibuziuk — APPROVED** [review]

**Comment by dkwon17** [comment]
Fixing failing CI check

**Pull request merged**